core/src/main/scala/org/apache/spark/scheduler/cluster/StandaloneSchedulerBackend.scala

```diff
@@ -100,7 +100,9 @@ private[spark] class StandaloneSchedulerBackend(
     val sparkJavaOpts = Utils.sparkJavaOpts(conf, SparkConf.isExecutorStartupConf)
     val javaOpts = sparkJavaOpts ++ extraJavaOpts
     val command = Command("org.apache.spark.executor.CoarseGrainedExecutorBackend",
-      args, sc.executorEnvs, classPathEntries ++ testingClassPath, libraryPathEntries, javaOpts)
+      args, sc.executorEnvs, classPathEntries ++ testingClassPath, libraryPathEntries
```

Comment:
please put the comma at the end of this line instead of the beginning of the next one.

Reply:
done